### PR TITLE
Login: Create View with Email and Password Fields

### PR DESCRIPTION
**Pull request by dhjiang1**
No Login button functionality yet and need to route to backend still

Screenshot 2023-03-12 at 8 13 36 PM

### CODE REVIEW

File: kidsability/src/containers/Login.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@media all and (min-width: 480px) {
+    .Login {
+      padding: 60px 0;
+    }
+  
+    .Login form {
+      margin: 0 auto;
+      max-width: 320px;
+    }
+  }
```
Comment (DaniilBacalear) [inline kidsability/src/containers/Login.css]: I think we should separate the css files from the js files. So maybe create a separate folder called styles and put all the css files there.

File: kidsability/src/containers/Login.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import "./Login.css";
+
+export default function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  function validateForm() {
+    return email.length > 0 && password.length > 0;
```
Comment (DaniilBacalear) [inline kidsability/src/containers/Login.js]: can use this library to do the validation https://www.npmjs.com/package/react-email-validator

File: kidsability/src/containers/Login.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            onChange={(e) => setPassword(e.target.value)}
+          />
+        </Form.Group>
+        <Button block="true" size="lg" type="submit" disabled={!validateForm()}>
+          Login
+        </Button>
+      </Form>
+    </div>
+  );
+}
```
Comment (DaniilBacalear) [inline kidsability/src/containers/Login.js:47]: Added some comments other then that LGTM

**Review by DaniilBacalear — APPROVED** [review]

File: kidsability/src/containers/Login.css (hunk 1 above)
Comment (dhjiang1) [reply kidsability/src/containers/Login.css]: Sounds good

File: kidsability/src/containers/Login.js (hunk 2 above)
Comment (dhjiang1) [reply kidsability/src/containers/Login.js]: validateForm checks to make sure both aren't null; i added the email-validate to the logic. Lmk what you think

File: kidsability/src/containers/Login.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import Button from "react-bootstrap/Button";
+import "../styling/Login.css";
+
+export default function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  // include check for valid email with @ character
+  function validateForm() {
+    var validate = require("react-email-validator")
```
Comment (DaniilBacalear) [inline kidsability/src/containers/Login.js]: put this after the imports and change var to const, I think going forward we should avoid using var altogether and instead use either let or const

**Pull request merged**